src/components/ThemeSelector.tsx

```diff
@@ -0,0 +1,31 @@
+import { useState } from "react";
+import { Helmet } from "react-helmet";
+
+// should be the same as names of the files in /public/themes
+const themes = ["light", "dark"] as const;
+type SupportedThemes = (typeof themes)[number];
+
+function ThemeSelector(): JSX.Element {
+  const [selectedTheme, setSelectedTheme] = useState<SupportedThemes>("light");
+
+  return (
+    <>
+      <Helmet>
+        <link rel="stylesheet" href={`themes/${selectedTheme}.css`} />
```

Comment:
This entire implementation is a hack for now, but I wonder what you think of this particular way of dealing with changing the stylesheet names @marcobiedermann :)

Reply:
This is not hacky at all. Looks good to me